.github/workflows/twister.yaml

```diff
@@ -240,10 +240,11 @@ jobs:
         if: always()
         uses: actions/upload-artifact@v4
         with:
-          name: HTML Unit Test Results
+          name: HTML & XML Unit Test Results
```

Comment:
how about just "Unit test results" or "Unit test result artifacts"?

Reply:
hey sure, sounds great :) 
didn't put a lot of thought into it tbh and didn't realize this was actually meant to be the name of the file. Latest push also fixes the actual name of the step